src/h3/_cy/CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ find_package(PythonExtensions MODULE REQUIRED)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
+# Build Cython annotations
+set(CYTHON_ANNOTATE TRUE)
```

Comment:
I don't think this is the case, but just to double check with you: This shouldn't slow down the finally built code in any way?

Any chance this slows down the other build paths?

Reply:
As in, would it make sense to add a condition that we only build the annotation html files when running the annotation CI workflow.

Reply:
I believe all it does is copy the Cython source and generated C code into a pretty HTML file.

As such, I would assume it has:

- no effect on _runtime code_
- no effect on installation time from a wheel (since these wouldn't be packaged in the wheel)
- slightly longer to compile from source

Reply:
Sounds good!